smart-contracts/contracts/mixins/MixinTransfer.sol

```diff
@@ -174,29 +137,18 @@ contract MixinTransfer is
   }
 
   /**
-   * @notice An ERC-20 style transfer.
-   * @param _value sends a token with _value * expirationDuration (the amount of time remaining on a standard purchase).
-   * @dev The typical use case would be to call this with _value 1, which is on par with calling `transferFrom`. If the user
-   * has more than `expirationDuration` time remaining this may use the `shareKey` function to send some but not all of the token.
+   * @notice An ERC-721 style transfer.
+   * @param _tokenId the Id of the token to send
+   * @param _to the destination address
+   * @return success bool success/failure of the transfer
    */
   function transfer(
```

Comment:
So let's keep the "value" and make it call `shareKey` because this is what it does in practice. Transfers time off of a single membership. ie: 1 => transfer the whole time, 0.5 transfer 50% of it... etc.

Reply:
but then wont that defeat the purpose of having both `shareKey` and `transfer`? What is the goal of having both?